files/image_config/process-reboot-cause/process-reboot-cause

```diff
@@ -22,6 +23,9 @@ REBOOT_CAUSE_DIR = "/host/reboot-cause/"
 REBOOT_CAUSE_FILE = REBOOT_CAUSE_DIR + "reboot-cause.txt"
 PREVIOUS_REBOOT_CAUSE_FILE = REBOOT_CAUSE_DIR + "previous-reboot-cause.txt"
 FIRST_BOOT_PLATFORM_FILE = "/tmp/notify_firstboot_to_platform"
+REBOOT_CAUSE_KEXEC = "/proc/cmdline"
```

Comment:
From /proc/cmdline, we determine reboot _type_, not necessarily reboot _cause_. I feel like this constant is better named something like `REBOOT_TYPE_KEXEC_FILE` and the constant on the next line should be named `REBOOT_TYPE_KEXEC_PATTERN`.

Reply:
will fix it

Reply:
fixed